core/src/main/java/com/scalar/db/api/AbacAdmin.java

```diff
@@ -339,51 +339,48 @@ default Optional<UserTagInfo> getUserTagInfo(String policyName, String username)
   }
 
   /**
-   * Applies the given policy to the given namespace.
+   * Creates a namespace policy with the given policy and the given namespace.
    *
+   * @param namespacePolicyName the namespace policy name
    * @param policyName the policy name
    * @param namespaceName the namespace name
    * @throws ExecutionException if the operation fails
    */
-  default void applyPolicyToNamespace(String policyName, String namespaceName)
+  default void createNamespacePolicy(
```

Comment:
Renamed `applyPolicyToNamespace` to `createNamespacePolicy`.

Also, changed the signature of the `enableNamespacePolicy`, `disableNamespacePolicy `, and `getNamespacePolicy` methods to specify the namespace policy name.